### ISSUE TITLE
Error in rule_nospace_line_end.py

### ISSUE THREAD

**Issue by jankraing** [priv-kweihmann/oelint-adv#413]
Correct me if I'm wrong but I think these lines:

https://github.com/priv-kweihmann/oelint-adv/blob/9c5695f399a26ac347b07fab471228f485dd9720/oelint_adv/rule_base/rule_nospace_line_end.py#L31-L32

Should look like this:

```python
i[0].RealRaw = RegexRpl.sub(r'\s{1,}\n', '\n', i[0].RealRaw)
i[0].Raw = RegexRpl.sub(r'\s{1,}\n', '\n', i[0].Raw) 
```

1. The variable i needs to be indexed or an error is thrown: `AttributeError: 'tuple' object has no attribute 'RealRaw'`
2. The substitution only removes two or more trailing white-spaces, a single trailing white-space remains.

**Comment by priv-kweihmann** [comment]
You're absolutely right :sweat_smile: - there's  major bug in testing, I'm currently fixing - expect a new release later that day
